### PR TITLE
Create Fact “field-thistle”

### PR DESCRIPTION
**Pull request by NicolasRobertson**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/facts/field-or-pasture-thistle.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  alt: Field or Pasture Thistle
+  creator: Tom Potterfield
+source:
+  name: "Bees: An Identification and Native Plant Forage Guide"
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+category: Plant
+---
+An important forage source for all pollinators, field thistle is often mistaken for invasive thistle species, and removed from land.
+
+Native thistle attracts a multitude of bees including leaf-cutter, bumble, long-horned, sweat and “small” sweat bees.
```
Comment (floogulinc) [inline src/facts/field-or-pasture-thistle.md]:
```suggestion
An important forage source for all pollinators, field thistle is often mistaken for invasive thistle species, and removed from land. Native thistle attracts a multitude of bees including leaf-cutter, bumble, long-horned, sweat and “small” sweat bees.
```

File: src/facts/field-or-pasture-thistle.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Field or Pasture Thistle
+image:
+  url: https://www.flickr.com/photos/tgpotterfield/14359215078
+  license: CC by-NC-SA 2.0
```
Comment (floogulinc) [inline src/facts/field-or-pasture-thistle.md]:
```suggestion
  license: CC BY-NC-SA 2.0
```

File: src/facts/field-or-pasture-thistle.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  src: ../images/pasturethistle.jpg
+  alt: Field or Pasture Thistle
+  creator: Tom Potterfield
+source:
+  name: "Bees: An Identification and Native Plant Forage Guide"
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+category: Plant
```
Comment (floogulinc) [inline src/facts/field-or-pasture-thistle.md]: I think we want all category names to be lowercase. Also do we want to have this "plant" category or does it fit "habitat" better?

**Review by floogulinc — CHANGES_REQUESTED** [review]

File: src/facts/field-or-pasture-thistle.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: Field or Pasture Thistle
```
Comment (floogulinc) [inline src/facts/field-or-pasture-thistle.md]: Can we just call this "Field Thistle"

**Review by floogulinc — CHANGES_REQUESTED** [review]

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to just `field-thistle.md`

File: src/facts/field-or-pasture-thistle.md (hunk 2 above)
Comment (floogulinc) [reply src/facts/field-or-pasture-thistle.md]: Please do this suggestion.
```suggestion
  license: CC BY-NC-SA 2.0
```

**Review by floogulinc — CHANGES_REQUESTED** [review]
Please rename the file to just `field-thistle.md`

**Review by kklamberty — APPROVED** [review]

**Review by floogulinc — APPROVED** [review]

**Pull request merged**